x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -2833,13 +2833,6 @@
         }
       }
     },
-    "cloud": {
```

Comment:
Resolving conflicts got me into this mess of re-sorting the plugins listed... no schemas have been changed other than the addition of `cloudExperiments`